### DISCUSSION TITLE
Is there any support for external links?

### DISCUSSION THREAD

**Discussion by gluiz1** [ahrm/sioyek#1147] [Q&A] [answered]
I have a lot of PDF's that uses links generated by [xr-hyper](https://ctan.org/pkg/xr-hyper) and [url](https://ctan.org/pkg/url) packages from LaTeX. But still now, I am unable to open these link by click on them. 

Is there a way to open these links or `sioyek` does not support these linking now?

**Comment by ahrm** [comment, marked as answer]
You mean symbolic links? In my testing they just work (at least on the development branch).

Edit: I get it now, you mean external links to other documents, we don't support that now but we will in the next version.

**Reply by gluiz1** [reply]
Glad to know! This will be a killer feature! 
Also, where is the documentation for the symbolic links? I did not find them.

**Reply by ahrm** [reply]
There is nothing special about symbolic links, they should just work like any normal file.
